estate/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': 'Estate',
+    'version': '0.0',
```

Comment:
```suggestion
    'version': '1.0',
```
Default is usually starting at 1.0.